### PR TITLE
WIP add article on local state management from apollo

### PR DESCRIPTION
**Pull request by KizeRemi**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
33 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, d'executer, d'iinjecter, enum, éxecute, familié, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, paraitre, readQuery, redux, Redux, réquêtes, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Comment by ElevenWilson** [comment]
35 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'executer, enum, Etant, éxecute, familié, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, paraitre, readQuery, redux, Redux, réquêtes, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Comment by ElevenWilson** [comment]
32 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'éxecuter, enum, Etant, éxecute, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, readQuery, redux, Redux, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Comment by ElevenWilson** [comment]
32 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'éxecuter, enum, Etant, éxecute, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, readQuery, redux, Redux, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Comment by ElevenWilson** [comment]
32 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'éxecuter, enum, Etant, éxecute, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, readQuery, redux, Redux, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Comment by ElevenWilson** [comment]
32 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'éxecuter, enum, Etant, éxecute, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, readQuery, redux, Redux, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Comment by ElevenWilson** [comment]
32 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'éxecuter, enum, Etant, éxecute, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, readQuery, redux, Redux, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

File: _posts/fr/2019-01-31-apollo-local-state-une-alternative-a-redux-contextapi.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+export  default resolvers;
+```
+Le principe est relativement simple, on écrit directement dans le cache avec la méthode writeData. Notre input contient les paramètres envoyés à la mutation qui vont nous permettre de créer et insérer l'objet Notification dans notre cache.
+La propriété **__typename** est obligatoire et nécessaire pour Apollo, il faut lui spécifier le nom de l'objet dans votre Schéma.
+Il est également possible de lire le cache avec le méthode readQuery si vous avez besoin des données précédentes avant la mutation. Par exemple, pour ajouter une nouvelle entrée à une liste.
+
+Vous remarquerez qu'il n'y a pas de resolver pour la query Notification, en effet, Apollo se charge de la retrouver automatiquement.
+
+### Les requêtes
+Maintenant que nous avons fait notre schéma et nos resolvers, il ne reste plus qu'à créer les queries/mutations.
```
Comment (CaptainJojo) [inline _posts/fr/2019-01-31-apollo-local-state-une-alternative-a-redux-contextapi.md]: un saut de ligne

**Comment by ElevenWilson** [comment]
32 words might be misspelled, please check them: apollo, apparait, apparaitre, contextapi, ContextApi, ContextAPI, d'Apollo, DevTools, d'éxecuter, enum, Etant, éxecute, forçément, graphql, GraphQL, HOC, intégre, js, lang, meetups, newNotification, readQuery, redux, Redux, resolvers, rmavillaz, stateLink, TTL, typeDefs, typename, withNotification, writeData

**Pull request merged**